### DISCUSSION TITLE
Adding New Datasets to the Package

### DISCUSSION THREAD

**Discussion by 000Justin000** [pyg-team/pytorch_geometric#2138] [Ideas]
Hello,

We recently created a new dataset for predicting county-level election outcomes during the 2012 & 2016 presidential elections. It has been used in the following papers:

Residual Correlation in Graph Neural Network Regression. KDD 2020
Combining Label Propagation and Simple Models Out-performs Graph Neural Networks. ICLR 2021
A Unifying Generative Model for Graph Learning Algorithms: Label Propagation, Graph Convolutions, and Combinations. arXiv

We would like to add this dataset to the package. What's the procedure here?

Thanks!
Junteng

**Comment by rusty1s** [comment]
Thanks for your interest in contributing. In general, it is sufficient to make a PR with the dataset placed in `torch_geometric.datasets`. You can find guidance in creating PyG datasets [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/create_dataset.html) or by looking at already integrated datasets, e.g., this [one](https://github.com/rusty1s/pytorch_geometric/blob/master/torch_geometric/datasets/reddit2.py). Let me know if you have any further questions.
